share/spack/templates/container/Dockerfile

```diff
@@ -1,3 +1,8 @@
+{% if bootstrap.image %}
+{{ bootstrap.recipe }}
+
+{% endif %}
+{% if render_build_phase %}
```

Comment:
can this be a property or something instead of a hard-coded thing?  e.g `if render_phase.build` is easier to produce from the command line arg I mentioned above.